keyboards/keyap/info.json

```
@@ -0,0 +1,45 @@
+{
+    "manufacturer": "PenEngineering",
+    "keyboard_name": "Keyapad",
+    "maintainer": "PenEngineering",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "nkro": true,
+       "encoder": true
```

No encoder configuration? The feature should not be enabled then.
```suggestion
        "nkro": true
```